frontend/components/Meetups/MeetupsTable.vue

```diff
@@ -1,52 +1,115 @@
 <script setup>
-import { CalendarIcon, ChevronRightIcon, MapPinIcon } from '@heroicons/vue/24/outline'
+import { ref, computed, defineProps } from 'vue'
+import { CalendarIcon, ChevronRightIcon, MapPinIcon, ChatBubbleLeftIcon, MagnifyingGlassIcon  } from '@heroicons/vue/24/outline'
 
-defineProps({
+const props = defineProps({
   meetups: {
     type: Array,
     default: () => [],
   },
 })
+
+const searchMeetup = ref('')
+const searchPresentation = ref('')
+const searchSpeaker = ref('')
+
+const searchSpeakers = ( meetups ) => {
+  return meetups.filter(meetup => meetup.presentations.some(({ speakers }) => 
+  speakers.some(({ name }) => 
+  name.toLowerCase().includes(searchSpeaker.value.toLowerCase()),
+    ),
+  ))
```

Comment:
There's something wrong with indentation here. Please check that.

Reply:
indentation should be good now. 
Design I will change in #10, is that ok?

Reply:
Okok.